x/msgfees/types/events.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func NewEventMsgs(totalCalls map[string]uint64, totalFees map[string]sdk.Coin) *EventMsgFees {
+	events := make([]EventMsgFee, len(totalCalls))
+	i := 0
```

Comment:
#nit instead of using this `i` counter, go allows you can initialize the events array to have a size of `len(totalCalls)` but start off at index 0 so you can use append to add to the length of the array.  Here is an example of that: https://go.dev/play/p/lkgNLF_EXqc

So this could look like:
```golang
events := make([]EventMsgFee, 0, len(totalCalls))

for typeURL, count := range totalCalls {
		total := totalFees[typeURL]
		events = append(events, EventMsgFee{
			MsgType: typeURL,
			Count:   fmt.Sprintf("%v", count),
			Total:   total.String(),
		})
	}
```

Reply:
I ended up changing this a bit.  I didn't like the idea of iterating through a map.  Therefore, I ended up sorting the map's keys and removing any keys that might not be contained in both maps.